ordec/sim2/ngspice_subprocess.py

```diff
@@ -82,46 +91,66 @@ def __init__(self, p: Popen, debug: bool, cwd: Path):
         self.p: Popen[bytes] = p
         self.debug = debug
         self.cwd = cwd
-        self._async_running = False
-        self._async_thread: Optional[threading.Thread] = None
         self._async_queue: Optional[queue.Queue] = None
-        self._async_halt_requested = False
+        self._async_thread: Optional[threading.Thread] = None
         self._async_lock = threading.Lock()
-        self._data_points_sent = 0
+        self._async_halt_requested = False
+        self._async_resume_event = threading.Event()
         self._async_current_time = 0.0
+        self._data_points_sent = 0
+        self._last_vector_length = 0
+        self._is_running = False
+        self._wrdata_file: Optional[Path] = None
+        self._wrdata_last_row = 0
+        self._wrdata_vectors: list[str] = []
+
+        self._configure_precision()
+
+    def _configure_precision(self) -> None:
+        """Configure ngspice numeric precision settings."""
+
+        try:
+            if self.debug:
+                _debug("Configuring ngspice numeric precision")
+            # Increase the number of digits printed in tabular outputs.
```

Comment:
I found a way to make ngspice-cli backend have the same precission as the ffi backend, so there is no need to distinguish them anymore